python/lsst/ts/cbp/csc.py

```diff
@@ -80,10 +83,15 @@ async def telemetry(self):
 
         """
         while True:
-            self.log.debug("Begin sending telemetry")
-            await self.component.update_status()
-            if self.component.status.panic:
-                self.fault(1, "CBP Panicked. Check hardware and reset device.")
+            try:
+                self.log.debug("Begin sending telemetry")
+                await self.component.update_status()
+                if self.component.status.panic:
+                    self.fault(1, "CBP Panicked. Check hardware and reset device.")
```

Comment:
I gently suggest adding a `return` after `self.fault(...)` to make it clear that the telemetry loop will not continue. It's not essential because going into FAULT state kills the the telemetry loop (eventually) but it makes sure that the telemetry loop cannot possibly run again while transitioning to fault state.